.husky/commit-msg

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn commitlint --edit $1
+npx commitlint --edit $1
```

Comment:
**question:** Why switch from `yarn` to `npx`?

Reply:
https://github.com/typicode/husky/issues/1246

This is the fix for the available version of node that we have on the self service